src/components/Book.css

```diff
@@ -0,0 +1,132 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;500;700&display=swap');
+
+:root {
+  --font: 'Roboto+Slab', sans-serif;
+}
+
+.book {
+  display: flex;
+  justify-content: space-between;
+  background-color: white;
+  padding-left: 15px;
+  padding-right: 75px;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  margin-bottom: 15px;
+  font-family: var(--font);
+  font-weight: 200;
+  border: #dfdfdf solid 1px;
+  box-shadow: 1px 1px 8px -5px #888;
+  border-radius: 5px;
+}
+
+.author {
+  font-weight: 500;
+  font-size: 0.7rem;
+  font-style: normal;
+  margin: 0 !important;
```

Comment:
- [ ] It would be better to avoid using `!important` a lot. Using this out of laziness will give you a headache later on as your style sheet will be harder to maintain. Using the !important rule is often considered a bad practice due to the clutter it can bring with one of CSS’s core mechanisms which is specificity.

*The !important rule was conscripted in the mid-to-late 90’s to help web designers and developers override the rules of their style sheet. The !important rule prevails over all the previous styles and says, “I’m more important, forget about all of that other CSS and use me!“.*